contracts/seed/Seed.sol

```diff
@@ -44,6 +44,7 @@ contract Seed {
     bytes public metadata; // IPFS Hash
```

Comment:
Overall comment, is described issue actually a bug?
As I understand the default decimals of the project token is 10^18, no?